pria_lifechem/models/test_demo.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import pandas as pd
+import csv
+import numpy as np
+import json
+import keras
+import sys
+import math
+from keras.models import Sequential, Model
+from keras.layers import Dense, Dropout, Input, Dense, Activation
+from keras.layers.normalization import BatchNormalization
+from keras.optimizers import SGD, Adam
+from sklearn.cross_validation import StratifiedShuffleSplit
+from pria_lifechem.function import read_merged_data, extract_feature_and_label, store_data, \
+    transform_json_to_csv, reshape_data_into_2_dim
+from pria_lifechem.evaluation import roc_auc_single, roc_auc_multi, bedroc_auc_multi, bedroc_auc_single, \
+    precision_auc_multi, precision_auc_single, enrichment_factor_multi, enrichment_factor_single
+from pria_lifechem.models.CallBacks import KeckCallBackOnROC, KeckCallBackOnPrecision, \
+    MultiCallBackOnROC, MultiCallBackOnPR
+from sklearn.ensemble import RandomForestClassifier
+
+
+# this is for customized loss function
+epsilon = 1.0e-9
+
+
+class DNN_RF:

```

Comment:
Why is this called `DNN_RF` instead of `RF`?  Is there a network being used here, or it just part of the legacy code from before.  Removing the `DNN` from the name may prevent confusion if anyone reads this.